pyproject.toml

```diff
@@ -29,11 +25,42 @@ exclude = [".git", "__pycache__", ".tox", "build", "dist", "*.egg_info", "venv"]
 select = ["E", "W", "F", "C", "N", "R", "D", "H"]
 # Ignore W503, E501 because using black creates errors with this
 # Ignore D107 Missing docstring in __init__
-ignore = ["W503", "E501", "D107"]
+ignore = ["W503", "E501", "D100", "D101", "D107", "D205", "D415"]
```

Comment:
Some of those are easy to fix by simply addressing the comments provided by the linter. It's not nice to go and ignore all the tests ones that complain.

Reply:
Addressed.